views/login.handlebars

```diff
@@ -16,27 +16,6 @@
       </div>
     </form>
   </div>
-  <div class="col-md-5">
-    <h2>Sign-up</h2>
-
-    <form class="form signup-form">
-      <div class="form-group">
-        <label for="new_user_name">User Name:</label>
-        <input class="form-input" type="text" id="new_user_name" />
-      </div>
-      <div class="form-group">
-        <label for="new_email">E-mail:</label>
-        <input class="form-input" type="text" id="new_email" />
-      </div>
-      <div class="form-group">
-        <label for="new_password">Password:</label>
-        <input class="form-input" type="password" id="new_password" />
-      </div>
-      <div class="form-group">
-        <button class="btn btn-primary" type="submit">Sign-up</button>
-      </div>
-    </form>
-  </div>
 </div>
 
 <script src="./js/login.js"></script>
```

Comment:
Again, this displays an error in the console because there is no signup button for the event listener.

Reply:
will create a signup.js like above